### PR TITLE
Bump Upbound Provider AWS to v1.6.1

### PR DESCRIPTION
**Pull request by artych**
Bump Upbound Provider AWS to v1.6.1
This PR should add v1beta2 CRDs

URL validation: https://doc.crds.dev/github.com/crossplane-contrib/provider-upjet-aws@v1.6.1

### CODE REVIEW

**Review by cindy — APPROVED** [review]

**Review by jvrplmlmn — APPROVED** [review]

File: libs/crossplane/config.jsonnet [hunk 1]
```diff
@@ -88,9 +88,9 @@ config.new(
     // Upbound official providers
     // https://marketplace.upbound.io/
     {
-      output: 'upbound-provider-aws/0.40',
+      output: 'upbound-provider-aws/1.6.1',
       prefix: '^io\\.upbound\\.aws\\..*',
-      crds: ['https://doc.crds.dev/raw/github.com/upbound/provider-aws@v0.40.0'],
+      crds: ['https://doc.crds.dev/raw/github.com/crossplane-contrib/provider-upjet-aws@v1.6.1'],
```
Comment (Duologic) [inline libs/crossplane/config.jsonnet:93]: The change to `crossplane-contrib` is odd.

Comment (Duologic) [reply libs/crossplane/config.jsonnet:93]: Oh, seems they moved it back to that org :|

Comment (Duologic) [reply libs/crossplane/config.jsonnet:93]:
```suggestion
      crds: ['https://doc.crds.dev/raw/github.com/crossplane-contrib/provider-upjet-aws@v1.6.0'],
```

This should work, not sure why the 1.6.1 doesn't work but the diff is not very relevant afaict: https://github.com/crossplane-contrib/provider-upjet-aws/compare/v1.6.0...v1.6.1

File: libs/crossplane/config.jsonnet [hunk 2]
```diff
@@ -88,9 +88,9 @@ config.new(
     // Upbound official providers
     // https://marketplace.upbound.io/
     {
-      output: 'upbound-provider-aws/0.40',
+      output: 'upbound-provider-aws/1.6.1',
```
Comment (jvrplmlmn) [inline libs/crossplane/config.jsonnet:91]: Note that if we have changed the version in CRDs (line 93), we should update this one to match:

```suggestion
      output: 'upbound-provider-aws/1.6.0',
```

**Review by jvrplmlmn — APPROVED** [review]
👍

**Pull request merged**